server/plugins/router/router.module.js

```diff
@@ -130,6 +131,33 @@ internals.registerRoutes = function(server, next) {
                 },
             },
         },
+        {
+            method: ['GET', 'POST'],
+            path: internals.paths.graphQL,
+            handler: {
+                proxy: {
+                    mapUri: function (req, cb) {
+                        return cb(
+                            null,
+                            `${internals.options.storeUrl}${req.path}`,
+                            Object.assign(
```

Comment:
I originally had this as a new arrow function with a spread operator, but this made tests fail due to a bug in [eslint/espree](https://github.com/eslint/espree/pull/426) which was only fixed a few months ago. Upgrading our version of espree by 4 major versions seemed prohibitively complicated to make this one change, so I think we should just stick with the weird syntax for now.